WooCommerce/Classes/Authentication/Epilogue/StorePickerViewController.swift

```diff
@@ -291,6 +291,15 @@ private extension StorePickerViewController {
             return
         }
 
+        // If a site address was passed in credentials, select it
+        // TODO: improve it somehow.
+        //
+        let siteAddress = StoresManager.shared.sessionManager.defaultCredentials?.siteAddress
+        for site in sites where site.url == siteAddress {
+            currentlySelectedSite = site
+            return
+        }
```

Comment:
@mindgraffiti you could simplify this code a little bit by doing something like:

```swift
// If a site address was passed in credentials, select it
if let siteAddress = StoresManager.shared.sessionManager.defaultCredentials?.siteAddress,
    let site = sites.filter({ $0.url == siteAddress }).first {
      currentlySelectedSite = site
      return
}
```

Reply:
Nice! Thanks @bummytime.